source/app/scorearea.cpp

```diff
@@ -140,6 +140,7 @@ void ScoreArea::renderDocument(const Document &document)
     }
 
     myScene.addItem(myCaretPainter);
+    refreshZoom();
```

Comment:
We want the document's zoom applied. Most likely there's a better way to integrate it in the rendering itself (in the Caret). I'd prefer to avoid breaking something with this oftenly used functionality.

Reply:
Is this needed because the document might now have a zoom level other than 100% after it's first created / rendered?

I think that's fine to call here - it just sets the overall transform for the scene, so it could probably even be done at the start of this method.

Reply:
Yep, the document didn't got rescaled with the initial value. I'll give it a try moving it to the 🔝